### ISSUE TITLE
Cannot revoke myca1

### ISSUE THREAD

**Issue by plusFeldt** [xipki/xipki#286]
Installation of xipki (using the released zip-File) and ca setup  (following the README.md) worked fine. myca1 has been created successfully. Trying to revoke it doesn't seem to work:

```
karaf@root()> ca:ca-revoke myca1
Error executing command ca:ca-revoke: option --reason is required
karaf@root()> ca:ca-revoke --reason keyCompromise myca1
Error executing command: Failed making field 'java.time.Instant#seconds' accessible; either increase its visibility or write a custom TypeAdapter for its declaring type.
karaf@root()> ca:ca-revoke --reason 1 myca1
Error executing command: Failed making field 'java.time.Instant#seconds' accessible; either increase its visibility or write a custom TypeAdapter for its declaring type.
karaf@root()>
```

Question: What's the correct xipki command/process to revoke the ca? Thx!

**Comment by xipki** [comment]
This error will be fixed in the 6.3.1.
You use the correct command to revoke the CA. What is the reason to revoke this CA?

**Comment by plusFeldt** [comment]
Many thanks for your feedback and the upcoming fix. My xipki installation is just for testing, the reason I've chosen was "keyCompromise" (see the 2nd command's error message). I tried ca:ca-revoke's --help parameter, hoping to find an example or hint how to specify the revocation reason (eg. as string or numeric), without success.
I'm experienced regarding x509 certificates, ca-hierarchies, their creation and usage, but I've no experience with xipki. yet. xipki just came to my attention, while searching for a powerful standalone ocsp-responder. Am I right, that the main xipki usage documentation is reading the various karaf scripts, est.sh and rest.sh? If so, that's ok, but some kind of written xipki-beginner's usage guide (á la "These are the top-10-karaf-commands to create, a basic two-level ca including ocsp-responder-, and e.g. a server certificate") would be very nice, too. ;-) A document similar to your easy to understand and follow installation/setup-README.md, that contains the steps/commands and a short explanation could flatten the learning curve enormously.
